src/__tests__/lib/item-input/array.test.ts

```diff
@@ -647,13 +653,18 @@ describe('checkboxDef', () => {
 			expect(stringFromUnknownMock).toHaveBeenCalledTimes(2)
 			expect(stringFromUnknownMock).toHaveBeenCalledWith('item1')
 			expect(stringFromUnknownMock).toHaveBeenCalledWith('item2')
-			expect(clipToMaximumMock).toHaveBeenCalledTimes(1)
-			expect(clipToMaximumMock).toHaveBeenCalledWith('Item 1, Item 2', maxItemValueLength)
+			expect(clipToMaximumMock)
+				.toHaveBeenCalledExactlyOnceWith('Item 1, Item 2', maxItemValueLength)
 		})
 
 		it('uses given summarizeForEdit', async () => {
-			const summarizeForEdit = jest.fn<Required<CheckboxDefOptions<string>>['summarizeForEdit']>()
-			const def = checkboxDef<string>('Checkbox Def', ['Item 1', 'Item 2', 'Item 3'], { summarizeForEdit })
+			const summarizeForEdit =
+				jest.fn<Required<CheckboxDefOptions<string>>['summarizeForEdit']>()
+			const def = checkboxDef(
```

Comment:
Same as above: why is `<string>` not used anymore?

Reply:
Same answer as before but I should also say that I usually leave these types off when they can be inferred. There are some other cases in this same file where the type can't be inferred and I left it on.